quinn-proto/src/endpoint.rs

```diff
@@ -389,6 +392,22 @@ where
         ConnectionEvent(ConnectionEventInner::NewIdentifiers(ids, now))
     }
 
+    /// Returns the destination IP used in outgoing transmit packets.
+    ///
+    /// Returns `None` if no specific IP should be used
+    fn transmit_dst_ip(&self, local_ip: Option<IpAddr>) -> Option<IpAddr> {
+        if self
```

Comment:
Same here. The `transmit_` prefix seems superfluous here, since it's pretty much implied by `dst`?

Reply:
Yes we can remove it, and I did that. However I then noticed that `dst_ip` is wrong, since this is the source IP address which is used for transmits, instead of the destination. We can't make it `local_ip`, because the function is already taken and doesn't take the setting into account (unless we drop the setting). Any other recommendation?

Reply:
`src_ip()`, then? Or the full `send_from_ip()`?

Reply:
Went for `src_ip()`